### DISCUSSION TITLE
Why Unknown S29GL limited to 4MB in code?

### DISCUSSION THREAD

**Discussion by StanMOI** [sanni/cartreader#994] [Need help]
Hello Sanni,

Just me make it clear for me.
Why you limited "Unknown S29GL type" to 4MB ? `flashSize = 4194304;`
Why it cannot be 8MB? `flashSize = 8388608;`

```
 // Unknown S29GL type
    else {
      println_Msg(F("Unknown S29GL Type"));
      flashSize = 4194304
      sectorSize = 65536;
      bufferSize = 32;
    }
```

**Comment by sanni** [comment]
I had just stopped working on this back then. Ideally now a menu would pop-up where you can select flashSize, sectorSize and bufferSize like it does for N64 repros with unknown S29GL type.
